src/test/java/com/wikia/webdriver/testcases/mobile/HTMLTitleTests.java

```diff
@@ -82,14 +82,14 @@ public void seo_guard() {
       PrintWriter out2 = new PrintWriter("./logs/seo-guard2.txt");
 
       for (String[] testCase : testCases) {
-        navigate.toUrl(new Page(testCase[0], testCase[1]).getUrl());
+        navigate.toUrl(new Page(testCase[0], testCase[1], testCase[2]).getUrl());
         new Wait(driver).forElementVisible(By.cssSelector(".side-nav-toggle-2016"));
         this.pushMetaTagsToFile(out);
 
         if (!originalEnv.equals("prod")) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the "prod" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1132)